importer/writer.py

```diff
@@ -26,13 +27,61 @@ def copy_files(self, source: str):
             if read.reverse_read is not None:
                 copyfile("%s/%s" % (source, read.reverse_read), "%s/%s" % (self.destination, read.reverse_read))
 
+    def download_files_from_ena(self,connections):
+        reads_to_download=[read.forward_read for read in self.spreadsheet.reads if not self.check_if_file_downloaded(read.forward_read)]
+        df = self.create_dataframe_from_list(reads_to_download)
+        df = create_commands(df,connections, self.destination)
+        submit_commands(df)
+
+    def create_dataframe_from_list(self,reads):
+        df = pd.DataFrame(([read, 'import_%s' % read] for read in reads),
+                          columns=('Read accession', 'Job_name'))
+        return df
+
+    def check_if_file_downloaded(self, accession):
+        single_ended_file = self.destination + '/' + accession + '.fastq.gz'
+        forward_file= self.destination + '/' +  accession + '_1.fastq.gz'
+        reverse_file= self.destination + '/' + accession + '_2.fastq.gz'
+        print(self.destination)
+        if path.exists(single_ended_file):
+            return True
+        elif path.exists(forward_file) and path.exists(reverse_file):
+                return True
+        else:
+            return False
+
     def save_workbook(self, workbook):
         workbook.save(self.spreadsheet_file)
 
     def create_destination_directory(self):
-        if path.isdir(self.destination) == False:
+        if not path.isdir(self.destination):
             mkdir(self.destination)
 
+
+def create_commands(df, connections, destination):
```

Comment:
Wonder whether it's worth to have a command line option to do the downloading without LSF. Would also make the Docker image more self-consistent. Nothing urgent but could think about it.